Doc/faq/library.rst

```diff
@@ -159,7 +159,7 @@ The "global main logic" of your program may be as simple as ::
 
 at the bottom of the main module of your program.
 
-Once your program is organized as a tractable collection of functions and class
+Once your program is organized as a tractable collection of function and class
```

Comment:
I think this section is okay as it is. I can see where you were going with this though, you might have assumed that the "and" was implying: "... collection of functions behaviors and class behaviors" (which would be grammatically incorrect). But, in this case the two are entirely separate.

Reply:
In that case, I don't understand why the doc states that "you should write test functions that exercise the behaviours". If "behaviours" only applies to "class" and not to "function", then the doc would say that functions not embedded into classes should not be tested. Am I missing something here ?

Reply:
> If "behaviours" only applies to "class" and not to "function", then the doc would say that functions not embedded into classes should not be tested,

Hmm, good point. I might've been looking at this correction too much in isolation without fully considering the surrounding context. While reading over that sentence a couple of times, I also realized it's missing a comma:

```
Once your program is organized as a tractable collection of function and class
behaviours, you should write test functions that exercise the behaviours.
```

Also an interesting tidbit (this is definitely subjective though, so I wouldn't suggest changing it for this PR), but we actually alternate between the American English and British English versions of "behaviors/behaviours" within the documentation:

```bash
$ git grep -E "behaviors" -- "*.rst" | wc -l
26
$ git grep -E "behaviours" -- "*.rst" | wc -l
8
```

Reply:
Ah nice catch (never occured to me there were two versions of this word). I've added the comma.